package.json

```diff
@@ -20,14 +20,39 @@
   "author": "shamofu",
   "license": "MIT",
   "build": {
+    "productName": "shamofu",
+    "appId": "org.shamofu.electronReactParcelBoilerplate",
     "files": [
       "app/**",
       "!app/main/main.ts",
       "!app/main/tests/**",
       "!app/renderer/**",
       "app/renderer/.parcel/production/**",
       "!app/**/*.map"
-    ]
+    ],
+    "mac": {
+      "target": [
+        "zip"
+      ],
+      "artifactName": "${productName}-mac.${ext}"
+    },
+    "win": {
+      "target": [
+        "zip"
+      ],
+      "artifactName": "${productName}-win.${ext}"
+    },
+    "linux": {
+      "target": [
+        "zip"
+      ],
+      "category": "Development",
+      "artifactName": "${productName}-linux.${ext}"
+    },
+    "directories": {
+      "buildResources": "resources",
+      "output": "release"
```

Comment:
Then [this line](https://github.com/shamofu/electron-react-parcel-boilerplate/blob/d9ff5abc63ca86449cea4d22f0b48fda52dc47c4/.gitignore#L2) can be removed.